tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -5710,6 +5710,30 @@ def test_bf16_2gpu_mtp_ar(self):
                 f"Acceptance rate too low for prompt {i}: {accept_rate:.2f}"
 
 
+class TestQwen3_5_4B(LlmapiAccuracyTestHarness):
+    MODEL_NAME = "Qwen/Qwen3.5-4B"
+    MODEL_PATH = f"{llm_models_root()}/Qwen3.5-4B"
+    EXTRA_EVALUATOR_KWARGS = dict(
+        apply_chat_template=True,
+        fewshot_as_multiturn=True,
+        chat_template_kwargs=dict(enable_thinking=False),
+    )
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)